api.swagger.yaml

```diff
@@ -1151,10 +2044,10 @@ components:
             https://demo.invotra.com/uuid/node/1f0c6cb5-a3c3-4197-bebc-b3e44462dc26
         short_text:
           description: >-
-            Contextual text for the feed message, with placeholders @user for the user's
-            name and optionally @group for the group name.
+            Contextual text for the feed message, which displays in between the
+            user's name and the entity title
           type: string
-          example: "@user created the following within @group"
+          example: created the following within
```

Comment:
Did you mean to change these or was it a copy/paste error?

Reply:
updated